lightkurve/lightcurve.py

```diff
@@ -129,7 +129,9 @@ def fold(self, period, phase=0.):
             A new ``LightCurve`` in which the data are folded and sorted by
             phase.
         """
-        fold_time = ((self.time - phase + 0.5 * period) / period) % 1 - 0.5
+        fold_time = (((self.time - ((phase) * period)) / period) % 1)
```

Comment:
also, no need for parenthesis between ``phase``, i.e.,
``(((self.time - ((phase) * period)) / period) % 1) ---> (((self.time - phase * period) / period) % 1) ``